src/chrome/content/rules/Apple.xml

```diff
@@ -534,6 +534,10 @@
 			<test url="http://www.apple.com/shop/payment_plan" />
 			<test url="http://www.apple.com/uk/shop/buy-mac/macbook/space-grey-512gb" />
 
+		<exclusion pattern="^http://www\.apple\.com/apple\-events/" />
```

Comment:
The backslash shouldn't be necessary.

Please document that the redirect is JS based.